receipt_dynamo/tests/integration/test__lock_manager_production_scenario.py

```
@@ -0,0 +1,283 @@
+            print(f"validate_ownership() on expired lock completed in {elapsed:.2f}s, result: {is_owner}")
+            # Could be True or False depending on exact timing
+        except TimeoutError:
+            signal.alarm(0)
+            elapsed = time.time() - start_time
+            manager.release()
+            pytest.fail(f"validate_ownership() on expired lock hung for {elapsed:.2f}s")
+        
+        try:
+            manager.release()
+        except:
+            pass  # May fail if lock expired
```

Ruff (0.12.2)

Use `contextlib.suppress(BaseException)` instead of `try`-`except`-`pass`

(SIM105)